EIPS/eip-7532.md

```diff
@@ -0,0 +1,40 @@
+---
+eip: 7532
+title: Generation unit (64 epochs)
+description: A Generation is a scale version of an epoch.
+author: JKincorperated (@JKincorperated)
+discussions-to: https://ethereum-magicians.org/t/another-eip-a-new-unit-the-generation/16024
+status: Draft
+type: Informational
+created: 2023-10-08
+---
+
+## Abstract
+
+This EIP proposes the introduction of a new terminology, a "generation" to be used alongside the existing "epoch" terminology in the Ethereum network. A generation will be defined as a period consisting of 64 epochs, with each epoch consisting of 32 slots. This terminology enhancement aims to provide a more granular way to describe time intervals within the Ethereum network.
+
+e.g. Slot 7497186 is in the 234287th epoch which is in the 3660th generation.
+
+## Motivation
+
+As the ethereum network matures and throughput and age increases larger measurement units are required. The generation is based off of the average time to activate a validator on the beacon chain and can be used to show long term time measurements.
```

Comment:
```suggestion
As the ethereum network matures and throughput and age increases larger measurement units are required. The generation is based off of the average time to activate a validator on the beacon chain and can be used to show long term time measurements, such as...
1. 
2. 
3.
```

Also, would 64 epochs a little too small if the goal is to allow for longer measurement?

Reply:
I originally thought about 128 epochs, however, each generation is around ~6 hours and is a good marker for each quarter of the day.